myhoard/controller.py

```diff
@@ -1168,9 +1179,16 @@ def _purge_old_binlogs(self, *, mysql_maybe_not_running=False):
                 last_binlog_purge=last_purge,
                 last_could_have_purged=last_could_have_purged,
             )
+
             self.stats.gauge_float("myhoard.binlog.time_since_any_purged", time.time() - last_purge)
             self.stats.gauge_float("myhoard.binlog.time_since_could_have_purged", time.time() - last_could_have_purged)
```

Comment:
Should pass extra tag like `"support_should_have_purged": "1"` here so that it is easier to switch alerting to alert on either this or the new one depending on what is available.

Reply:
added a tag